core/src/test/java/org/infinispan/api/flags/FlagsEnabledTest.java

```diff
@@ -93,6 +93,8 @@ public void testWithFlagsSemantics() {
             cache1LocalOnly.withFlags(SKIP_CACHE_LOAD);
       MagicKey localKey2 = new MagicKey("local2", cache1);
       cache1SkipRemoteAndStores.put(localKey2, "value");
+      // local-mode operation does not get into PrepareCommand as modification and WSC is not executed
+      // (though, it's committed as it's in context and it's stored as it is retured by localTransaction.getAllModifications()
```

Comment:
How about "CACHE_MODE_LOCAL operation is not replicated with the PrepareCommand, but the entry is updated on the origin"?